src/partition/repart.c

```diff
@@ -4866,6 +4869,20 @@ static int context_acquire_partition_uuids_and_labels(Context *context) {
                                 return r;
                 }
 
+                /* Derive the verity salt and verity superblock UUID from the seed to keep them reproducible */
+                if (p->verity == VERITY_HASH) {
+                        const char salt_token[] = "verity-salt";
```

Comment:
We don't need to define a variable for this. Please just use a define:
```C
#define VERITY_SALT "verity-salt"
```
and then use `VERITY_SALT` and `strlen(VERITY_SALT)` in the call.

Reply:
Even the define is not needed if we add `derive_salt()` like I proposed.